e2e/test-component/scenarios/embedding-sdk/editable-dashboard.cy.spec.tsx

```diff
@@ -12,12 +12,12 @@ describeEE("scenarios > embedding-sdk > editable-dashboard", () => {
   beforeEach(() => {
     signInAsAdminAndEnableEmbeddingSdk();
 
-    cy.createDashboard(
-      {
-        name: "Embedding SDK Test Dashboard",
-      },
-      { wrapId: true },
-    );
+    cy.createDashboard({
+      name: "Embedding SDK Test Dashboard",
+    }).then(({ body: dashboard }) => {
+      cy.wrap(dashboard.id).as("dashboardId");
+      cy.wrap(dashboard.entity_id).as("dashboardEntityId");
```

Comment:
All existing test setups are changed so that we manually set aliases, so I could also set the entity ID alias which I'll need to use for testing entity IDs